docs/runtimes/create-and-use-functions.ipynb

```diff
@@ -5,6 +5,7 @@
    "id": "8d3e3a9c",
```

Comment:
<blockquote>MLRun adds, by default, "nuclio-"  </blockquote><p>I'm assuming this is only relevant to nuclio functions</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mlrun/mlrun/pull/4756/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
updated